pkg/console/operator/operator.go

```diff
@@ -157,6 +177,24 @@ func (c *consoleOperator) handleSync(operatorConfig *operatorsv1.Console, consol
 		// handled below
 	case operatorsv1.Unmanaged:
 		logrus.Println("console is in an unmanaged state.")
+		v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
+			Type:    operatorsv1.OperatorStatusTypeAvailable,
+			Status:  operatorsv1.ConditionUnknown,
+			Reason:  "Unmanaged",
```

Comment:
Shall we make a `const` for the `Reason` `"Unmanaged"`?  To reuse rather than copy/paste.